src/ser/mod.rs

```diff
@@ -191,6 +193,16 @@ impl PrettyConfig {
 
         self
     }
+
+    /// Configures whether the extensions are included in the output as
```

Comment:
I think we should describe the use-case here and what to look out for (enabling the same extensions when deserializing).